t/alignment/algx.xml

```diff
@@ -8,128 +8,168 @@
+          <XMath>
+            <XMDual>
+              <XMApp>
+                <XMTok meaning="list"/>
+                <XMRef idref="alg1.l1.m1.1"/>
+                <XMRef idref="alg1.l1.m1.2"/>
+              </XMApp>
+              <XMWrap>
+                <XMTok font="italic" role="UNKNOWN" xml:id="alg1.l1.m1.1">a</XMTok>
+                <XMTok role="PUNCT">,</XMTok>
+                <XMTok font="italic" role="UNKNOWN" xml:id="alg1.l1.m1.2">b</XMTok>
+              </XMWrap>
+            </XMDual>
+          </XMath>
+        </Math>)<text cssstyle="float:right"><Math mode="inline" tex="\triangleright" text="triangleright" xml:id="alg1.l1.m2">
+            <XMath>
+              <XMTok name="triangleright" role="ADDOP">▷</XMTok>
+            </XMath>
+          </Math> The g.c.d. of a and b
+</text></listingline>
+      <listingline xml:id="alg1.l2"><tags>
+          <tag><text fontsize="80%">2:</text></tag>
+          <tag role="refnum">2</tag>
+        </tags>     <Math mode="inline" tex="r\leftarrow a\bmod b" text="r leftarrow modulo@(a, b)" xml:id="alg1.l2.m1">
           <XMath>
             <XMApp>
-              <XMTok fontsize="80%" name="leftarrow" role="ARROW">←</XMTok>
-              <XMTok font="italic" fontsize="80%" role="UNKNOWN">r</XMTok>
+              <XMTok name="leftarrow" role="ARROW">←</XMTok>
+              <XMTok font="italic" role="UNKNOWN">r</XMTok>
               <XMApp>
-                <XMTok fontsize="80%" meaning="modulo" name="bmod" role="MODIFIEROP">mod</XMTok>
-                <XMTok font="italic" fontsize="80%" role="UNKNOWN">a</XMTok>
-                <XMTok font="italic" fontsize="80%" role="UNKNOWN">b</XMTok>
+                <XMTok meaning="modulo" name="bmod" role="MODIFIEROP">mod</XMTok>
+                <XMTok font="italic" role="UNKNOWN">a</XMTok>
+                <XMTok font="italic" role="UNKNOWN">b</XMTok>
               </XMApp>
             </XMApp>
           </XMath>
-        </Math><text fontsize="80%">
-</text></listingline>
-      <listingline refnum="3" xml:id="alg1.l3"><tag><text fontsize="80%">3:</text></tag><text fontsize="80%">      </text><text font="bold" fontsize="80%">while</text><text fontsize="80%"> </text><Math mode="inline" tex="r\not=0" text="r not-equals 0" xml:id="alg1.l3.m1">
+        </Math>
+</listingline>
+      <listingline xml:id="alg1.l3"><tags>
+          <tag><text fontsize="80%">3:</text></tag>
+          <tag role="refnum">3</tag>
+        </tags>     <text font="bold">while</text> <Math mode="inline" tex="r\not=0" text="r not-equals 0" xml:id="alg1.l3.m1">
           <XMath>
             <XMApp>
-              <XMTok fontsize="80%" meaning="not-equals" name="not-=" role="RELOP">≠</XMTok>
-              <XMTok font="italic" fontsize="80%" role="UNKNOWN">r</XMTok>
-              <XMTok fontsize="80%" meaning="0" role="NUMBER">0</XMTok>
```

Comment:
unsure if this is minor, but the fontsize attribute disappeared from the individual math tokens? Seems unrelated as a change.

Reply:
I think this was due to a bugfix that came about as a side-effect of treating tags more uniformly: sometimes the formatter for a tag was something like `\bf\thesomething`; Hopefully the current code is more consistent about wrapping a group around so that the sideeffects don't leak out, like the small font did here.